wiki/Beatmapping/Mapping_techniques/Making_good_sliders/en.md

```diff
@@ -81,11 +81,11 @@ Courtesy of mm201.
 The number of control points you want depends on the angle covered by your arc:
 
 - 0 degrees: 2 points.
-- 0 degrees - 20 degrees: 3 points.
-- 20 degrees - 170 degrees: 4 points.
-- 170 degrees - 200 degrees: 5 points.
-- 200 degrees - 300 degrees: 6 points.
-- 300 degrees - 350 degrees: 7 points.
+- 0 degrees–20 degrees: 3 points.
+- 20 degrees–170 degrees: 4 points.
+- 170 degrees–200 degrees: 5 points.
+- 200 degrees–300 degrees: 6 points.
+- 300 degrees–350 degrees: 7 points.
```

Comment:
same as other comment but not as bad. for this one I think it just looks cleaner in context of line 83
```suggestion
- 0–20 degrees: 3 points.
- 20–170 degrees: 4 points.
- 170–200 degrees: 5 points.
- 200–300 degrees: 6 points.
- 300–350 degrees: 7 points.
```